
device.h caues major pain for the user-space, so do not include it ;-)

  - Rene Rebe <rene@exactcode.de>

--- include/linux/genhd.h.orig	2004-10-14 16:07:26.910854072 +0200
+++ include/linux/genhd.h	2004-10-14 16:07:04.280294440 +0200
@@ -12,7 +12,7 @@
 #include <linux/config.h>
 #include <linux/types.h>
 #include <linux/major.h>
-#include <linux/device.h>
+/*#include <linux/device.h>*/
 #include <linux/smp.h>
 #include <linux/string.h>
 #include <linux/fs.h>
